### PR TITLE
Update small promo to use resizer v2

### PR DESCRIPTION
**Pull request by matthewroach**
## Description

Update Small Promo to use Resizer

## Jira Ticket

- [THEMES-736](https://arcpublishing.atlassian.net/browse/THEMES-736)

## Test Steps

1. Checkout this branch `git checkout THEMES-736-small-promo-resizer-update`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/small-promo-block`
3. Add the small promo block to a page and configure a content source
4. Validate the Image is being rendered via resizer v2

## Author Checklist

- [x] Confirmed all the test steps a reviewer will follow above are working.
- [x] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [x] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [x] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [x] Confirmed this PR has unit test files
  - [x] Ran `npm run test`, made sure all tests are passing
  - [x] If the amount of work to write unit tests for this change are excessive,
        please explain why (so that we can fix it whenever it gets refactored).
- [x] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

### CODE REVIEW

File: blocks/small-promo-block/features/small-promo/default.jsx [hunk 1]
```diff
@@ -37,7 +48,7 @@ const SmallPromo = ({ customFields }) => {
 			// does not need embed_html because no video section
 			// does not need website section nor label because no overline
 			// does not need byline because no byline shown
-			filter: `{
+			filters: `{
```
Comment (nschubach) [inline blocks/small-promo-block/features/small-promo/default.jsx]: This now seems to be pulling back a ton more information where just filter seemed to be more specific and related to the promo...

filter: 
image

filters:
image

Comment (nschubach) [reply blocks/small-promo-block/features/small-promo/default.jsx]: I assume this was a test of some sort as this looks like it's not even applying the filter structure.

Comment (matthewroach) [reply blocks/small-promo-block/features/small-promo/default.jsx]: Fixed

**Review by nschubach — APPROVED** [review]
All issues resolved and look good to me.

**Comment by matthewroach** [comment]
@nschubach I am merging this - no need for design review as we are putting back the images going through resizer

**Pull request merged**